### ISSUE TITLE
Bug in Templates?

### ISSUE THREAD

**Issue by Skeeve** [kovetskiy/mark#197]
I had two macros referencing files "command.start" and "command.end".

Unfortunately the last one didn't work. Instead the first one was always applied.

I found the reason in this line:

https://github.com/kovetskiy/mark/blob/5428cc683302a7679901e23d4e97fc83838a788b/pkg/mark/includes/templates.go#L29

Is there any reason for using just the filename without path and extension as the name?

**Comment by kovetskiy** [comment]
Not really, we can remove this operation. It's mostly for dropping `.template` or other file extensions.

**Comment by Skeeve** [comment]
> Not really, we can remove this operation. It's mostly for dropping `.template` or other file extensions.

Will you do?
